modules/holodeckb2b-common/pom.xml

```diff
@@ -80,6 +80,17 @@
             <artifactId>commons-io</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```

Comment:
This specific version of log4j is probably needed for use with the MockAppender? It would be good if we can change this, so the log4j version as managed by the main pom can be used here. This however does not need to be solved in this PR. It is just something todo, maybe just add a note here for now?